src/v2/Components/NavBar/Menus/NotificationsMenu.tsx

```diff
@@ -42,87 +76,103 @@ export const NotificationMenuItems: React.FC<NotificationsMenuQueryResponse> = p
 
   return (
     <>
-      {notifications.map(({ node }, index) => {
-        const { artists, href, image, summary } = node
-        const worksForSaleHref = href + "/works-for-sale"
-        return (
-          <MenuItem
-            href={worksForSaleHref}
-            key={index}
-            onClick={() => {
-              handleClick(href, AnalyticsSchema.Subject.Notification)
-            }}
-          >
-            <Flex alignItems="center">
-              <Box width={40} height={40} bg="black5" mr={1}>
-                <Image
-                  src={image.resized.url}
-                  width={40}
-                  height={40}
-                  style={{
-                    objectFit: "cover",
-                  }}
-                  alt={summary}
-                />
-              </Box>
-              <Box>
-                <Sans size="2">{summary}</Sans>
-                <Sans size="2" weight="medium">
-                  {artists}
-                </Sans>
-              </Box>
-            </Flex>
-          </MenuItem>
-        )
-      })}
-
-      <Flex py={1} flexDirection="column" alignItems="center">
-        <>
-          {notifications.length === 0 && (
-            <Flex width="100%" flexDirection="column">
-              <Box pt={1} pb={3} width="100%" textAlign="center">
-                <Serif size="3">No new works</Serif>
-              </Box>
-            </Flex>
-          )}
-
-          <Box width="100%" px={2}>
-            <Separator />
-          </Box>
-
-          <Box pt={2}>
-            <Sans size="2">
-              <Link
-                aria-label="View all notifications"
-                href="/works-for-you"
-                onClick={() => {
-                  handleClick("/works-for-you", AnalyticsSchema.Subject.ViewAll)
-                }}
-              >
-                View all
-              </Link>
-            </Sans>
-          </Box>
-        </>
-      </Flex>
+      {notifications.map(
+        ({ node: { artists, href, image, summary } }, index) => {
+          return (
+            <MenuItem
+              href={`${href}/works-for-sale`}
+              key={index}
+              aria-label={`${summary} by ${artists}`}
```

Comment:
This is the label that gets announced — we ignore the underlying content and go with this.